src/transport/RendezvousSession.h

```diff
@@ -25,6 +31,28 @@
 
 namespace chip {
 
+/**
+ * RendezvousSession establish and maintain the first connection between
```

Comment:
nit: "establishes and maintains" or "is used to establish and maintain"

Reply:
Done.